src/pages/edge/adobe-journey-optimizer-decisioning/api-reference.md

```diff
@@ -92,6 +108,12 @@ For details on syntax, usage and availability, refer to [Mobile Core - Reset ide
 
 ## updatePropositions
 
+<InlineAlert variant="warning" slots="header, text1"/>
+
+This API has been deprecated starting in v3.2.2 and removed in v3.3.0 of the Android mobile extension.
```

Comment:
Did we not deprecate this in iOS? Can we add that also to the note?
Also we cannot remove APIs in minor version update. Can you change `and removed in v3.3.0 of the Android mobile extension.` to `They will be removed in the next major release of the Optimize SDK.`